cmd/report.go

```diff
@@ -286,17 +292,34 @@ func writeDetailedCSV(records []*reporting.CookbookRecord) {
 }
 
 func writeNodeReport(records []reporting.NodeReportItem) {
+
+	termWidth, _, err := terminal.GetSize(int(os.Stdout.Fd()))
+	if err != nil {
+		termWidth = MIN_TERM_WIDTH
+	}
+
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"Node Name", "Chef Version", "OS", "OS Version", "Cookbooks"})
-	table.SetReflowDuringAutoWrap(true)
-	table.SetRowLine(true)
 	table.SetAutoWrapText(true)
 	table.SetReflowDuringAutoWrap(true)
-	table.SetBorder(true)
+	table.SetColMinWidth(0, int(float64(termWidth)*0.30)) // fqdns can get pretty long
+	table.SetColMinWidth(1, int(float64(termWidth)*0.10)) // chef version is tiny
+	table.SetColMinWidth(2, int(float64(termWidth)*0.15)) // OS+version string
+	table.SetHeader([]string{"Node Name", "Chef Version", "Operating System", "Cookbooks"})
+	table.SetAutoFormatHeaders(false)
+	table.SetRowLine(false)
+	table.SetColumnSeparator(" ")
+	table.SetBorder(false)
 	for _, record := range records {
 		table.Append(record.Array())
 	}
+
+	fmt.Print("\n")
 	table.Render()
+	if termWidth < MIN_TERM_WIDTH {
+		fmt.Printf("\nNote: If the output above is not formatted correctly,")
```

Comment:
nit: I'm not sure about the word `Note:` here, maybe asking UX for feedback, or just display the message without that work? 💭

Reply:
I've tried it both ways, and it seemed that "Note:" worked better to differentiate the output from the actual report output.  I'll see if we can set aside some time to go through all of the outputs with UX, across commands.   I think there are probably a few areas for improvement. 

I've been trying to attach some screenshots with both formats, but github isn't cooperating in letting me attach after an upload failure earlier.  I'll come back later after I restart the browser...